projects/manager/src/app/datasets/loading-info/gdal-metadata-list/gdal-metadata-list.component.html

```diff
@@ -0,0 +1,82 @@
+<form [formGroup]="form">
+    <div *ngIf="dataPath">
+        <div *ngIf="dataPath" class="flex-container">
+            <mat-form-field appearance="fill">
+                <mat-label>Path to probe file, relative to upload or volume (optional)</mat-label>
+                <input matInput type="text" [(ngModel)]="mainFile" [ngModelOptions]="{standalone: true}" />
+            </mat-form-field>
+
+            <button mat-raised-button class="analyze" (click)="suggest()">Analyze file and suggest properties</button>
+        </div>
+
+        <mat-divider></mat-divider>
+    </div>
+
+    <div formGroupName="rasterResultDescriptor">
+        <mat-form-field appearance="fill">
+            <mat-label>Output Data Type</mat-label>
+            <mat-select formControlName="dataType">
+                <mat-option *ngFor="let dataType of RasterDataTypes" [value]="dataType">
+                    {{ dataType }}
+                </mat-option>
+            </mat-select>
+        </mat-form-field>
+
+        <mat-form-field appearance="fill">
+            <mat-label>Spatial Reference</mat-label>
+            <input matInput type="text" formControlName="spatialReference" />
+        </mat-form-field>
+
+        <mat-form-field appearance="fill">
+            <mat-label>Band Name</mat-label>
+            <input matInput type="text" formControlName="bandName" />
+        </mat-form-field>
+    </div>
+
+    <div class="container">
+        <mat-nav-list class="left">
+            <mat-list-item
+                *ngFor="let control of form.controls.timeSlices.controls; let i = index"
+                (click)="selectTimeSlice(i)"
+                [class.selected]="selectedTimeSlice === i"
+            >
+                <h4 matListItemTitle>
+                    {{ getTime(i) }}
+                </h4>
+                <p matListItemLine>{{ control.controls.gdalParameters.value.filePath }}</p>
+            </mat-list-item>
+            <mat-error *ngIf="form.controls.timeSlices.errors?.overlappingTimeInterval">Time slices must not overlap.</mat-error>
+            <div class="empty-placeholder" *ngIf="form.controls.timeSlices.controls.length === 0">
+                <i>No time slices defined.</i>
+            </div>
+            <button class="add-entry" mat-raised-button (click)="addTimeSlicePlaceholder()">Add Time Slice</button>
+            <button *ngIf="form.controls.timeSlices.controls.length > 0" class="add-entry" mat-raised-button (click)="removeTimeSlice()">
+                Remove selected time slice
+            </button>
+        </mat-nav-list>
+
+        @if (form.controls.timeSlices.controls.at(selectedTimeSlice); as selectedTimeSliceGroup) {
+            <div class="time-slice" [formGroup]="selectedTimeSliceGroup">
+                <geoengine-time-interval-input [formControl]="selectedTimeSliceGroup.controls.time"></geoengine-time-interval-input>
```

Comment:
The time input should default to intervals for rasters

Reply:
3c4815628baf841e00d094e22270b9eb7171dd42